Makefile

```diff
@@ -1,90 +1,120 @@
 exe = cmd/commander/*
 cmd = commander
-TRAVIS_TAG ?= "0.0.0"
+GIT_RELEASE_TAG ?= "0.0.0"
 CWD = $(shell pwd)
 
-.PHONY: deps lint test integration integration-windows git-hooks init
-
+.PHONY: init
 init: git-hooks
 
+.PHONY: git-hooks
 git-hooks:
 	$(info INFO: Starting build $@)
 	ln -sf ../../.githooks/pre-commit .git/hooks/pre-commit
 
+.PHONY: deps
 deps:
 	$(info INFO: Starting build $@)
 	go mod vendor
 
+.PHONY: build
 build:
 	$(info INFO: Starting build $@)
 	go build $(exe)
 
+.PHONY: lint
 lint:
 	$(info INFO: Starting build $@)
 	golint pkg/ cmd/
 
+.PHONY: test
 test:
 	$(info INFO: Starting build $@)
 	go test ./...
 
+.PHONY: test-coverage
 test-coverage:
 	$(info INFO: Starting build $@)
 	go test -coverprofile c.out ./...
 
+.PHONY: test-coverage-all-dockerized
+test-coverage-all-dockerized:
+	$(info INFO: Starting build $@)
+	./test.sh
+
+.PHONY: test-coverage-all-dockerized-with-codeclimate
+test-coverage-all-dockerized-with-codeclimate:
+	$(info INFO: Starting build $@)
+	CC_TEST_REPORTER_ID=${CC_TEST_REPORTER_ID} ./test.sh
 
 test-coverage-all: export COMMANDER_TEST_ALL = 1
 test-coverage-all: export COMMANDER_SSH_TEST = 1
-test-coverage-all: export COMMANDER_TEST_SSH_HOST = localhost:2222
+test-coverage-all: export COMMANDER_TEST_SSH_HOST = 172.28.0.2:22
 test-coverage-all: export COMMANDER_TEST_SSH_USER = root
 test-coverage-all: export COMMANDER_TEST_SSH_IDENTITY_FILE = $(CWD)/integration/containers/ssh/id_rsa
+.PHONY: test-coverage-all
 test-coverage-all:
 	$(info INFO: Starting build $@)
-	./integration/setup_unix.sh
 	go test -coverprofile c.out ./...
-	./integration/teardown_unix.sh
 
+test-coverage-all: export COMMANDER_TEST_ALL = 1
+test-coverage-all: export COMMANDER_SSH_TEST = 1
+test-coverage-all: export COMMANDER_TEST_SSH_HOST = 172.28.0.2:22
+test-coverage-all: export COMMANDER_TEST_SSH_USER = root
+test-coverage-all: export COMMANDER_TEST_SSH_IDENTITY_FILE = $(CWD)/integration/containers/ssh/id_rsa
+.PHONY: test-coverage-all-codeclimate
+test-coverage-all-codeclimate:
+	$(info INFO: Starting build $@)
+	./test-reporter before-build
+	go test -coverprofile c.out ./...; \
+	./test-reporter after-build -t gocov --prefix=github.com/commander-cli/commander --exit-code $$?
+
+.PHONY: integration-unix
 integration-unix: build
 	$(info INFO: Starting build $@)
 	commander test commander_unix.yaml
 
+.PHONY: integration-linux
 integration-linux: build
 	$(info INFO: Starting build $@)
-	./integration/setup_unix.sh
-	commander test commander_unix.yaml
-	commander test commander_linux.yaml
-	./integration/teardown_unix.sh
+	DOCKER_HOST=${DOCKER_HOST} DOCKER_CERT_PATH=${DOCKER_CERT_PATH} commander test commander_unix.yaml
```

Comment:
I can't quite figure out where these vars are coming from?

Reply:
They are coming from circle CI and are default docker env variables. 
If docker runs on a remote host it is necessary to provide these paths.